torch/testing/_internal/common_cuda.py

```diff
@@ -20,6 +20,7 @@
 
 SM53OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (5, 3))
 SM60OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (6, 0))
+SM70OrLater = LazyVal(lambda: torch.cuda.is_available() and torch.cuda.get_device_capability() >= (8, 0))
```

Comment:
should this be `and torch.cuda.get_device_capability() >= (7, 0))`? Also, is this used anywhere?

Reply:
It used to be used to gate torch.compile tests, but then I made it fallback to SHA-1 so that skip could be removed. Still, it seems useful to have.